gpcontrib/arenadata_toolkit/expected/adb_collect_table_stats_test.out

```diff
@@ -86,6 +100,58 @@ WHERE relname = 'db_files_history';
      2
 (1 row)
 
+-- Create a partition for the month before the current one
+SELECT add_extra_partition_to_db_files_history();
+ add_extra_partition_to_db_files_history 
+-----------------------------------------
+ 
+(1 row)
+
+-- There are "db_files_history" and three partitions
+-- (default, for the current month and for the month before the current one)
+SELECT count(inhrelid)
+FROM pg_inherits
+LEFT JOIN pg_class ON oid = inhparent
+WHERE relname = 'db_files_history';
+ count 
+-------
+     3
+(1 row)
+
+-- Remove the partition from "db_files_history" for the current month
+SELECT remove_partition_from_db_files_history();
+ remove_partition_from_db_files_history 
+----------------------------------------
+ 
+(1 row)
+
+-- There are "db_files_history" and two partitions (default and for the month before the current one)
+SELECT count(inhrelid)
+FROM pg_inherits
+LEFT JOIN pg_class ON oid = inhparent
+WHERE relname = 'db_files_history';
```

Comment:
What about "SELECT count(1) FROM pg_inherits WHERE inhparent='arenadata_toolkit.db_files_history'::regclass" ?

Reply:
The same query is used in the existing code of this test (refer to old code above in this file). So, if we change it here, we'll need to change it everywhere here in this file. And will be a refactoring, which we decided not to mix with the bugfix.

Reply:
As discussed, will fix this particular one. The queries in the existing code should be changed in a separate change for refactoring.